cmake/modules/LibnfcDrivers.cmake

```diff
@@ -3,13 +3,13 @@ SET(LIBNFC_DRIVER_ACR122_PCSC OFF CACHE BOOL "Enable ACR122 support (Depends on
 SET(LIBNFC_DRIVER_ACR122_USB ON CACHE BOOL "Enable ACR122 support (Direct USB connection)")
 SET(LIBNFC_DRIVER_ACR122S ON CACHE BOOL "Enable ACR122S support (Use serial port)")
 SET(LIBNFC_DRIVER_ARYGON ON CACHE BOOL "Enable ARYGON support (Use serial port)")
-IF(WIN32)
+IF(NOT UNIX OR APPLE)
```

Comment:
May I suggest to reverse the test and branches, to ease reading ?
if (unix or apple) ... set ON... else... set OFF

Reply:
I know it's ugly but there is no LINUX macro in cmake, and APPLE falls into UNIX like LINUX, i could do IF(UNIX AND NOT APPLE). Let me know

Reply:
damn, I managed to read it wrong myself :laughing: 
yes I tend to prefer UNIX AND NOT APPLE